app/settings/page.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import { useState } from 'react';
+import { Button, Divider, Spinner } from '@nextui-org/react';
+import { GoDownload } from 'react-icons/go';
+
+export default function Settings() {
+  const [loading, setLoading] = useState(false);
+
+  const handleDownload = () => {
+    setLoading(true);
+    // Simulate download delay
+    setTimeout(() => {
+      setLoading(false);
+      window.location.href = '/api/logs';
+    }, 2000);
```

Comment:
Was this just for testing or do we want it to behave this way?

Reply:
I know all nick wants is that spinner :)

Reply:
Ah yeah, meant to drop this, nice catch